.github/workflows/sycl-linux-build.yml

```diff
@@ -176,6 +182,9 @@ jobs:
     - name: check-llvm
       if: always() && !cancelled() && contains(inputs.changes, 'llvm')
       run: |
+        if [[ ${{ inputs.cc }} == 'icx' ]] || [[ ${{ inputs.cxx }} == 'icpx' ]]; then
+          export LIT_FILTER="SYCL"
```

Comment:
Can we please add a comment here why `LIT_FILTER` was required?

Reply:
i think its expected so will add comment, thanks